README.md

```diff
@@ -7,100 +9,92 @@
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=peterhuene/azure-functions-rs)](https://dependabot.com)
 [![license](https://img.shields.io/crates/l/azure-functions.svg)](https://github.com/peterhuene/azure-functions-rs/blob/master/LICENSE)
 
-This is an early-stage framework for implementing [Azure Functions](https://azure.microsoft.com/en-us/services/functions/)
+An early-stage framework for implementing [Azure Functions](https://azure.microsoft.com/en-us/services/functions/)
 in [Rust](https://www.rust-lang.org/).
 
-## Disclaimer
-
-Although the maintainer of this project is a Microsoft employee, this project is not an officially recognized Microsoft product and is not an endorsement of any future product offering from Microsoft.
-
-_Microsoft and Azure are registered trademarks of Microsoft Corporation._
+> :triangular_flag_on_post: **Disclaimer**  
+> Although the maintainer of this project is a Microsoft employee, this project is not an officially recognized Microsoft product and is not an endorsement of any future product offering from Microsoft.  
+> _Microsoft and Azure are registered trademarks of Microsoft Corporation._
 
-## Example
+## Get Started
 
-A simple HTTP-triggered Azure Function:
+- [Examples](https://github.com/peterhuene/azure-functions-rs/tree/master/examples)
+- [Documentation](https://docs.rs/azure-functions/0.7.0/azure_functions/)
+- [Contributing](https://github.com/peterhuene/azure-functions-rs/blob/master/CONTRIBUTING.md)
 
-```rust
-use azure_functions::bindings::{HttpRequest, HttpResponse};
-use azure_functions::func;
-
-#[func]
-pub fn greet(req: HttpRequest) -> HttpResponse {
-    // Log the message with the Azure Functions Host
-    info!("Request: {:?}", req);
-
-    format!(
-        "Hello from Rust, {}!\n",
-        req.query_params().get("name").map_or("stranger", |x| x)
-    ).into()
-}
-```
+## Table of Contents
 
-See the [examples](https://github.com/peterhuene/azure-functions-rs/tree/master/examples) directory for the complete list of examples.
+- [Installation](#installation)
+  - [Requirements](#requirements)
+    - [CMake](#cmake)
```

Comment:
Very soon this and the one below are going away 😄.

![](https://media.giphy.com/media/S4P8Z5fiLRpOU/giphy.gif)

Reply:
Soon!

Reply:
If you rebase your feature branch, accept yours for the conflict resolution.  Let's remove the CMake and C++ compiler sections then.